### ISSUE TITLE
Cannot find type 'Font' in scope

### ISSUE THREAD

**Issue by appfrilans** [dufflink/vfont#4]
I get the following errror when exporting my app:

`Cannot find type 'Font' in scope`

On this row:

public extension Font {

in VFont.swift

Any ideas?

**Comment by dufflink** [comment]
Hello! Thank you for the issue.
Could you please tell me the iOS version? And what package manager did you use?

**Comment by appfrilans** [comment]
Hi,

I used SPM and latest iOS version (15.5)

Regards,
Jonas

**Comment by dufflink** [comment]
I've just created a new iOS project to test it. Works fine 🧐

Could you please kindly compare the attached zip file with your project?
[test-vfont.zip](https://github.com/dufflink/vfont/files/9082838/test-vfont.zip)

**Comment by dufflink** [comment]
@appfrilans Let me please know if it doesn't help 🙏

**Comment by appfrilans** [comment]
Testing your project and it seems to be working fine. Even with Catalyst. Will do some research and get back to you. Thanks!

**Comment by appfrilans** [comment]
Now it just works! Not sure what I did to make it work in my own project...

**Comment by appfrilans** [comment]
I take that back. Did not try exporting. Can even reproduce it with your test-project. Just click "Any iOS device (arm64)" as target and then Product > Archive and it will give you that error.

**Comment by dufflink** [comment]
You are right! I faced the same error when tried to Archive my project. I'll fix it as soon as possible

**Comment by dufflink** [comment]
@appfrilans Something strange happened. I installed Xcode 14 and now I can't repeat the previous scenario. 'Product > Archive' works. I tried to launch the same process in Xcode 13 and now it works too.
Could you please try to repeat the actions?
